webapp/dojo_news/src/components/NewsListItem.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div>
+    <h2>{{title}} ({{votes}})</h2>
+    <button @click="$emit('upvote')">Upvote</button>
+    <button @click="$emit('downvote')">Downvote</button>
+    <button @click="$emit('remove')">Remove</button>
+  </div>
+</template>
+
+<script>
+import {defineComponent} from 'vue';
+
+export default defineComponent({
+  name: "NewsListItem",
+  emits: ['upvote', 'downvote','remove'],
```

Comment:
Unfortunately, we'll downgrade back to Vue 2 since Storybook support is still not there for Vue 3 (also, `vue-test-utils` is still in beta for Vue 3, as is `vue-jest`)!

`emits` is a great addition with Vue 3 - but otherwise, the changes should not have too big of an impact.